### ISSUE TITLE
Can't seem to get mobile strategy

### ISSUE THREAD

**Issue by ssanchez** [sitespeedio/plugin-gpsi#18]
**USING:**
sitespeed.io 12.0.1-plus1 image (includes this and the lighthouse plugin)

Whenever I run I can see both plugins running, though the PSI score returned seems to be only for desktop. I am sending in --mobile as an option to no avail. I believe this is behavior similar to that reported in #5

**Comment by soulgalore** [comment]
Hi @ssanchez I wonder maybe if something changed in GPSI, I can have try later today.

**Comment by soulgalore** [comment]
I think the npm lib has changed and we missed to update on our side. https://developers.google.com/speed/docs/insights/v4/reference/pagespeedapi/runpagespeed we need to set the strategy (mobile|desktop). Any chance @ssanchez you can do a PR? One problem is that I couldn't see in the result that it returns the strategy (so we know that it works).

**Comment by soulgalore** [comment]
I was wrong., I  tried and for me it seems to work. Can you show me how it doesn't work?
Adding  `--mobile` when I test www.sitespeed.io gives different score (97 vs 90).

**Comment by ssanchez** [comment]
Perhaps I misunderstand what's happening. I'm testing a page whose mobile score differs dramatically in general than its desktop score:

https://www.business.com/articles/google-pay-vs-apple-pay-vs-samsung-pay/

When using the web-based PSI tool from Google I get low 30's for mobile and high 80's / low 90's for desktop. When I run the page through sitespeed's docker "plus-1" container I get high-80's for both (with and without the `--mobile` param)

`docker run --shm-size=2g --rm -v "$(pwd)":/sitespeed.io sitespeedio/sitespeed.io:12.0.1-plus1 https://www.business.com/articles/google-pay-vs-apple-pay-vs-samsung-pay/ --mobile`

Is this not apples-to-apples maybe?

**Comment by soulgalore** [comment]
hmm ok. I'm not sure how its implemented on GPSI side but I would have guess it would be apples to apples :) I tested and I git 88 vs 83 and I could see that I got different rules/summaries back.

So ... I tried https://github.com/GoogleChromeLabs/psi and that seems to be the same as the online tool. It seems that the old API hasn't been updated. 

I'll create an issue for updating the plugin to use the new one, but I wonder if it is just the same as running Lighthouse?
